src/main/java/com/minecolonies/coremod/client/gui/WindowMultiBlock.java

```diff
@@ -0,0 +1,292 @@
+package com.minecolonies.coremod.client.gui;
+
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.blockout.Log;
+import com.minecolonies.blockout.Pane;
+import com.minecolonies.blockout.controls.ButtonImage;
+import com.minecolonies.blockout.controls.TextField;
+import com.minecolonies.blockout.views.View;
+import com.minecolonies.coremod.MineColonies;
+import com.minecolonies.coremod.network.messages.MultiBlockChangeMessage;
+import com.minecolonies.coremod.tileentities.TileEntityMultiBlock;
+import net.minecraft.client.Minecraft;
+import net.minecraft.entity.player.EntityPlayer;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.entity.player.EntityPlayer'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)